pages/ksphere/konvoy/1.4/release-notes/index.md

```diff
@@ -100,6 +100,8 @@ enterprise: false
 
 #### Disclaimer
 
+-   Versions of Konvoy `v1.4.x` are not compatible and are not supported with [Kubernetes Base Addons](https://github.com/mesosphere/kubernetes-base-addons) `v2.x+` or [Kommander](https://d2iq.com/solutions/ksphere/kommander) `v1.1+`.
```

Comment:
This we'll need to backport in konvoy from master but i'll take care of that

Reply:
Thanks Sam!